api/v1beta1/kuadrant_types.go

```diff
@@ -19,6 +19,7 @@ package v1beta1
 import (
 	"github.com/go-logr/logr"
 	"github.com/google/go-cmp/cmp"
+	authorinov1beta1 "github.com/kuadrant/authorino-operator/api/v1beta1"
```

Comment:
Let's make sure we use the latest version of the `Authorino` API here: https://github.com/Kuadrant/kuadrant-operator/pull/561

Reply:
Once that PR is merged I will make sure to rebase to include it.

Reply:
Merged